code-challenges.js

```diff
@@ -10,17 +10,34 @@ var people = [
 ]
 // Expected output example: "Ford Prefect is a hitchhiker." "Zaphod Beeblebrox is a president of the galaxy." "Arthus Dent is a radio employee."
 
+const names = (obj) => {
+//   obj = obj.toLowerCase()
+//       .split(' ')
+//       .map((s) => s.charAt(0).toUpperCase() + s.substring(1))
+//       .join(' ');
+// return obj
 
+}
 
 
 
+console.log(names(people))
+
+
 // --------------------2) Create a function that takes in a mixed data array and returns an array of only the remainder of the numbers when divided by 3.
 
 var testingArray1 = [23, "Heyyyy!", 45, -9, 0, "Yo", false]
 // Expected output: [ 2, 0, -0, 0 ]
 var testingArray2 = [5, "Hola", 43, -34, "greetings", true]
 // Expected output: [ 2, 1, -1 ]
 
+const divideByThree = (arr1, arr2) => {
+  let bigArray = arr1.concat(arr2)
+  let array = [];
+   return (bigArray.filter(val => typeof val === "number"))
+}
+
```

Comment:
so this does return only numbers but we are missing some of the objectives of the prompt. I'm not sure why you've concatenated the two arrays? and we want to return the remainder of those numbers after they've been divided by 3 (HINT: %). You're totally on a good path though you'd just need some refactoring.